python/tritonserver/_api/wrapper.py

```diff
@@ -0,0 +1,953 @@
+            and not self.ready()
+            and ((timeout is None) or (time.time() - start_time) < timeout)
+        ):
+            time.sleep(polling_interval)
+        if not self.ready():
+            raise UnavailableError("Timeout before ready")
+
+    def stop(self) -> None:
+        self._server.stop()
+        self._server = Server._UnstartedServer()
+
+    def unregister_model_repository(self, repository_path: str) -> None:
+        self._server.unregister_model_repository(repository_path)
+
+    def register_model_repository(
+        self, repository_path: str, name_mapping: dict[str, str]
+    ) -> None:
+        """Add a new model repository.
+
+        Adds a new model repository.
+
+        See :c:func:`TRITONSERVER_ServerRegisterModelRepository`
+
+        Parameters
+        ----------
+        repository_path : str
+            repository path
+        name_mapping : dict[str, str]
+            override model names
+
+        Examples
+        --------
+        server.register_model_repository("/workspace/models",{"test_model":"new_model"})
+
+        """
+        name_mapping_list = [
+            _triton_bindings.TRITONSERVER_Parameter(name, value)
+            for name, value in name_mapping.items()
+        ]
+
+        self._server.register_model_repository(repository_path, name_mapping_list)
+
+    def poll_model_repository(self) -> None:
+        return self._server.poll_model_repository()
+
+    def metadata(self) -> dict[str, Any]:
+        return json.loads(self._server.metadata().serialize_to_json())
+
+    def get_metadata(self) -> dict[str, Any]:
+        return json.loads(self._server.metadata().serialize_to_json())
+
+    def live(self) -> bool:
+        return self._server.is_live()
+
+    def is_live(self) -> bool:
+        return self._server.is_live()
+
+    def ready(self) -> bool:
+        return self._server.is_ready()
+
+    def is_ready(self) -> bool:
+        return self._server.is_ready()
+
+    def model(self, model_name: str, model_version: int = -1) -> Model:
+        if isinstance(self._server, Server._UnstartedServer):
+            raise InvalidArgumentError("Server not started")
+        return Model(self._server, model_name, model_version)
+
+    def get_model(self, model_name, model_version=-1) -> Model:
+        if isinstance(self._server, Server._UnstartedServer):
+            raise InvalidArgumentError("Server not started")
+
+        return Model(self._server, model_name, model_version)
+
+    def models(self, exclude_not_ready: bool = False) -> ModelDictionary:
+        return ModelDictionary(self._server, self._model_index(exclude_not_ready))
+
+    def get_models(self, exclude_not_ready: bool = False) -> ModelDictionary:
+        return ModelDictionary(self._server, self._model_index(exclude_not_ready))
+
+    def model_index(self, exclude_not_ready: bool = False) -> ModelDictionary:
+        return ModelDictionary(self._server, self._model_index(exclude_not_ready))
```

Comment:
Let's only have a single method for each functionality (i.e., either use `live` or `is_live` but not both.